### DISCUSSION TITLE
Display Channel dB Value/Volume

### DISCUSSION THREAD

**Discussion by etonnote** [tttapa/Control-Surface#1028] [Q&A]
Hi,

I was wondering if there was a way to implement reading a channels current dB value and displaying it on an OLED screen? I have my control surface to a point where all of my controls are working, I am displaying track names, mutes, solos, rec arms, bank numbers etc. on the OLED but I cannot figure out if it is possible to display the track volume. 

I can see that I am receiving the mackie pitch bend values from ableton (via the channelMessageCallback) but I wouldn't know where to start in using that data and displaying it.

Any help would be greatly appreciated, thanks!

**Comment by tttapa** [comment]
You can use the [PBValue](https://tttapa.github.io/Control-Surface-doc/Doxygen/d2/d47/classPBValue.html) class.

To display such a value, you can get inspiration from the [VU meter display element code](https://tttapa.github.io/Control-Surface-doc/Doxygen/d0/db9/VUDisplay_8hpp_source.html).

**Reply by etonnote** [reply]
Perfect, thanks a lot!
